version_info.go

```diff
@@ -0,0 +1,306 @@
+package fileversion
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/xerrors"
+)
+
+// FixedFileInfo contains info from VS_FIXEDFILEINFO windows structure.
+type FixedFileInfo struct {
+	FileMajor      uint16
+	FileMinor      uint16
+	FileBuild      uint16
+	FilePrivate    uint16
+	ProductMajor   uint16
+	ProductMinor   uint16
+	ProductBuild   uint16
+	ProductPrivate uint16
+	FileFlagsMask  uint32
+	FileFlags      uint32
+	FileOs         uint32
+	FileType       uint32
+	FileSubType    uint32
+	FileDateMS     uint32
+	FileDateLS     uint32
+}
+
+type Locale string
```

Comment:
smt like 

```
// Locale defines a pair of a language ID and a codepage ID concatenated in a hexadecimal string.
// It can be either one of default locales or any of locales get from the file information using `.Locales()` method
```